### PR TITLE
chore(reports): async/await purchase analysis

### PR DESCRIPTION
**Pull request by jniles**
Migrates the purchase analysis report to async/await.  It also corrects several small details, such as comments, translation keys, and typos.

Here it is in action:
g1v31DWJTr

### CODE REVIEW

File: server/controllers/stock/reports/purchaseOrderAnalysis/index.js [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
 const TEMPLATE = './server/controllers/stock/reports/purchaseOrderAnalysis/report.handlebars';
-// expose to the API
 exports.report = report;
 
 // default report parameters
 const DEFAULT_PARAMS = {
-  csvKey : 'brea_report',
-  filename : 'TREE.BREAK_EVEN_REPORT',
+  csvKey : 'inventoriesOrdered',
+  filename : 'REPORTS.PURCHASE_ORDER_ANALYIS.TITLE',
```
Comment (jeremielodi) [inline server/controllers/stock/reports/purchaseOrderAnalysis/index.js]: this should be ```REPORTS.PURCHASE_ORDER_ANALYSIS.TITLE```

**Review by jeremielodi — APPROVED** [review]
There was just a typo but in general everything look okay to me

**Comment by jniles** [comment]
@jeremielodi thanks for the review.  I've fixed the typo.

**Pull request merged**